haskell/test-common/Lorentz/Contracts/Test/Management.hs

```diff
@@ -0,0 +1,557 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
+
+-- | Tests for management entrypoints of stablecoin smart-contract
+
+module Lorentz.Contracts.Test.Management
```

Comment:
Why is this module in `test-common`?

Reply:
Because `FA2.hs` was 🤷

Reply:
`FA2.hs` is used by both test-suites, `Managemenet.hs` is not.

Reply:
But it will be once I start working on nettest scenarios anyway

Reply:
Why do you think so? It has a large portion of `hspec` stuff. How do you plan to use it in nettest?

Reply:
That being said, `FA2.hs` is not used by nettest anymore either, so I guess it's time to move it to `test/` (as well as `Management.hs`).